src/Bookings.js

```diff
@@ -1,18 +1,45 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    let filteredBookings = [];
+    bookings.forEach(booking => {
+      if (
+        booking.firstName.toLowerCase().includes(searchVal.toLowerCase()) ||
+        booking.surname.toLowerCase().includes(searchVal.toLowerCase())
+      ) {
+        filteredBookings.push(booking);
+      }
```

Comment:
good that you're converting both the search val and booking to lowercase here :). `forEach` works, but as you are filtering the bookings, the `filter` method would work perfectly for this kind of task